### PR TITLE
TASK-45348 : Wallet New view

### PR DESCRIPTION
**Pull request by GouadriaHanen**

### CODE REVIEW

File: wallet-webapps-common/src/main/webapp/css/main.less [hunk 1]
```diff
@@ -730,6 +733,12 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
   .symbol{
     color : @errorColor;
   }
+  .balance{
+    color: #808080;
```
Comment (Rdidmohamedelhadi) [inline wallet-webapps-common/src/main/webapp/css/main.less]: use color variables

File: wallet-webapps/src/main/webapp/vue-app/components/wallet-app/Summary.vue [hunk 2]
```diff
@@ -15,7 +15,7 @@ along with this program; if not, write to the Free Software Foundation,
 Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 -->
 <template>
-  <v-flex id="walletSummary" class="elevation-0 me-3">
+  <v-flex id="walletSummary" class="elevation-0 me-3 ">
```
Comment (Rdidmohamedelhadi) [inline wallet-webapps/src/main/webapp/vue-app/components/wallet-app/Summary.vue]:
```suggestion
  <v-flex id="walletSummary" class="elevation-0 me-3">
```

File: wallet-webapps/src/main/webapp/vue-app/components/wallet-app/Summary.vue [hunk 3]
```diff
@@ -45,81 +45,96 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
-          </span>
-          <v-progress-circular
-            color="primary"
-            indeterminate
-            size="20" />
-        </v-badge>
-        <v-spacer />
-      </v-card-title>
-  
+    </template>
```
Comment (Rdidmohamedelhadi) [inline wallet-webapps/src/main/webapp/vue-app/components/wallet-app/Summary.vue]: template is empty, please delete it

File: wallet-webapps/src/main/webapp/vue-app/components/wallet-app/Summary.vue [hunk 4]
```diff
@@ -172,13 +189,26 @@ export default {
         return null;
       },
     },
+    displayWarnings: {
+      type: Object,
+      default: function() {
+        return null;
+      },
+    },
```
Comment (Rdidmohamedelhadi) [inline wallet-webapps/src/main/webapp/vue-app/components/wallet-app/Summary.vue]:
```suggestion
    displayWarnings: {
      type: Boolean,
      default: false,
    },
```

File: wallet-webapps/src/main/webapp/vue-app/components/wallet-app/Summary.vue [hunk 5]
```diff
@@ -172,13 +189,26 @@ export default {
...
+    displayDisapprovedWallet: {
+      type: Object,
+      default: function() {
+        return null;
+      },
+    },
```
Comment (Rdidmohamedelhadi) [inline wallet-webapps/src/main/webapp/vue-app/components/wallet-app/Summary.vue]:
```suggestion
    displayDisapprovedWallet: {
      type: Boolean,
      default: false
    },
```

File: wallet-webapps/src/main/webapp/vue-app/components/wallet-app/SummaryTransaction.vue [hunk 6]
```diff
@@ -15,29 +15,12 @@ along with this program; if not, write to the Free Software Foundation,
 Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 -->
 <template>
-  <v-card class="walletSummaryBalance headline elevation-3  pa-1 mt-2">
-    <v-card-title class="title subtitle-1 text-truncate pb-1" @click="displayTransactionList">
+  <div
+    class="border-box-sizing clickable addProduct">
```
Comment (Rdidmohamedelhadi) [inline wallet-webapps/src/main/webapp/vue-app/components/wallet-app/SummaryTransaction.vue]:
```suggestion
  <div class="border-box-sizing clickable addProduct">
```

**Review by Rdidmohamedelhadi — CHANGES_REQUESTED** [review]

**Review by hbenali — COMMENTED** [review]
You already submitted a commit https://github.com/Meeds-io/wallet/commit/219666318f9a51ede6dff980a71115558e5959c8 with the same name. Could you please submit a convenient PR title and description to let others understand what's going on on this PR (Just recommendation of best practice). Thanks !

File: wallet-webapps-common/src/main/webapp/css/main.less [hunk 7]
```diff
@@ -566,13 +566,18 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
     text-transform: capitalize;
+    margin: 0 -1%;
   }
   .walletSummaryBalance {
     display: inline-block;
     width: 100%;
     height: 100px;
+    margin: 0 1%;
     .title, .headline {
       color: #808080;
```
Comment (Rdidmohamedelhadi) [inline wallet-webapps-common/src/main/webapp/css/main.less]:
```suggestion
      color: @textMediumColor;
```

**Review by Rdidmohamedelhadi — CHANGES_REQUESTED** [review]

**Review by Rdidmohamedelhadi — APPROVED** [review]
LGTM

**Review by Rdidmohamedelhadi — APPROVED** [review]

**Pull request merged**